app/ParserTests/ParserTests.hs

```diff
@@ -82,11 +82,10 @@ fcesInputs = [
     , ("Completion of a minimum of 9 FCEs", FCES 9.0 $ REQUIREMENT $ RAW "")
     , ("Completion of 4.0 credits", FCES 4.0 $ REQUIREMENT $ RAW "")
     , ("at least 4.0 credits", FCES 4.0 $ REQUIREMENT $ RAW "")
-    , ("At least one 0.5 credit at the 400-level", FCES 0.5 $ REQUIREMENT $ RAW "the 400-level")
```

Comment:
Don't remove the tests. If they currently produce `DEPARTMENT` values, that's fine.